test/integration/trigger-updates.spec.ts

```diff
@@ -9,10 +10,11 @@ let ctx: testUtils.TestContext;
 
 beforeAll(async () => {
 	ctx = await testUtils.newContext();
+	await Bluebird.delay(1000);
```

Comment:
Why do we need to delay?

Reply:
Not a nice solution, but I couldn't find a fix

Reply:
Hmm, I know you can set a timeout as the second option to `beforeAll()`, maybe we should try that. I'll see if I can reproduce on my end and find a way to not use bluebird delay.